server/routes/document.js

```diff
@@ -1,12 +1,12 @@
 const router = require('express').Router();
-const { getOriginalDocument, getPdfDocument, getPdfDocumentPreview } = require('../middleware/document');
+const { getDocumentInfo, getOriginalDocument, getPdfDocument, getPdfDocumentPreview } = require('../middleware/document');
 
-router.get('/:caseId/document/:documentId/preview', getPdfDocumentPreview);
+router.get('/:caseId/document/:documentId/preview', getDocumentInfo, getPdfDocumentPreview);
 
-router.get('/:caseId/stage/:stageId/download/document/:documentId/pdf', getPdfDocument);
+router.get('/:caseId/stage/:stageId/download/document/:documentId/pdf', getDocumentInfo, getPdfDocument);
 
-router.get('/:caseId/stage/:stageId/download/document/:documentId/original', getOriginalDocument);
+router.get('/:caseId/stage/:stageId/download/document/:documentId/original', getDocumentInfo, getOriginalDocument);
 
 router.get('/:caseId/stage/:stageId/download/standard_line/:documentId', getOriginalDocument);
```

Comment:
This was an implicit decision to exclude from the permission check, as it doesn't contain any user information.